src/style/app.module.css

```
@@ -4188,3 +4189,1540 @@
+
+.dispflexUserPasswordUpdate {
+  display: flex;
+  justify-content: flex-start;
+  margin: 0 auto;
+}
+
+.dispbtnflex {
+  width: 90%;
+  margin-top: 20px;
+  display: flex;
+  margin: 0 30%;
```

Biome (1.9.4)

[error] Unexpected shorthand property margin after margin-top



(lint/suspicious/noShorthandPropertyOverrides)